llm_memory/service/memory_manager.py

```
@@ -678,16 +747,30 @@ async def _consolidate_previous_task_memory(self, current_task_id: str):
 
         Args:
             current_task_id: 当前新创建的任务记忆ID
+
+        Fix 5: 查询条件从 is_consolidated=False 改为 strength>0，
+        因为 is_consolidated 字段已通过 comprehensive_recall 迁移删除。
+        改用 strength>0 和 memory_type 过滤，确保查到所有活跃任务记忆。
         """
         try:
-            # 查找除了当前记忆之外的所有新鲜任务记忆
+            # 先获取当前任务的 memory_scope，防止跨范围误更新
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)